CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 1.4.30-M1 (Dev 2020)
+  * [KT-43597](https://youtrack.jetbrains.com/issue/KT-43597) Xcode 12.2 support
+  * [KT-43276](https://youtrack.jetbrains.com/issue/KT-43276) Add watchos_x64 target
+  * Support value classes
+  * [KT-43198](https://youtrack.jetbrains.com/issue/KT-43198) Init blocks inside of inline classes
+  * [KT-42649](https://youtrack.jetbrains.com/issue/KT-42649) Fix secondary constructors of generic inline classes
+  * [KT-38772](https://youtrack.jetbrains.com/issue/KT-38772) Support non-reified type parameters in typeOf
+  * Untying LLVM
```

Comment:
It's not about LLVM only. Let's call it "Improved compiler hackability" :)

Reply:
Hackability is a bad word. I would suggest "customization"